### PR TITLE
WebRTC chrome updates

### PR DESCRIPTION
**Pull request by a2sheppy**
Fix a couple version numbers for Chrome.

Sources:
* https://chromestatus.com/feature/5703036164898816
* https://www.chromestatus.com/feature/4749621767176192

### CODE REVIEW

File: api/RTCPeerConnection.json [hunk 1]
```diff
@@ -395,10 +395,10 @@
           "mdn_url": "https://developer.mozilla.org/docs/Web/API/RTCPeerConnection/addTransceiver",
           "support": {
             "chrome": {
-              "version_added": "69"
+              "version_added": "72"
```
Comment (jpmedley) [inline api/RTCPeerConnection.json]: Actually, this did land in 69.
https://chromium.googlesource.com/chromium/src/+/16b2a7fb36f48ea5f69d856aea2b244936140202%5E%21/#F70
https://storage.googleapis.com/chromium-find-releases-static/16b.html#16b2a7fb36f48ea5f69d856aea2b244936140202

File: api/RTCRtpTransceiver.json [hunk 2]
```diff
@@ -5,10 +5,10 @@
         "mdn_url": "https://developer.mozilla.org/docs/Web/API/RTCRtpTransceiver",
         "support": {
           "chrome": {
-            "version_added": false
+            "version_added": "72"
```
Comment (jpmedley) [inline api/RTCRtpTransceiver.json]: Transceiver itself landed in 69 https://storage.googleapis.com/chromium-find-releases-static/842.html#842893e6fdc0d81da782035ae314433d5c91df20

File: api/RTCRtpTransceiverDirection.json [hunk 3]
```diff
@@ -5,10 +5,10 @@
         "mdn_url": "https://developer.mozilla.org/docs/Web/API/RTCRtpTransceiverDirection",
         "support": {
           "chrome": {
-            "version_added": true
+            "version_added": "72"
```
Comment (jpmedley) [inline api/RTCRtpTransceiverDirection.json]: Also 69 https://storage.googleapis.com/chromium-find-releases-static/842.html#842893e6fdc0d81da782035ae314433d5c91df20

File: api/RTCRtpTransceiverInit.json [hunk 4]
```diff
@@ -5,10 +5,10 @@
         "mdn_url": "https://developer.mozilla.org/docs/Web/API/RTCRtpTransceiverInit",
         "support": {
           "chrome": {
-            "version_added": true
+            "version_added": "72"
```
Comment (jpmedley) [inline api/RTCRtpTransceiverInit.json]: 69
https://storage.googleapis.com/chromium-find-releases-static/16b.html#16b2a7fb36f48ea5f69d856aea2b244936140202

File: api/RTCTrackEvent.json [hunk 5]
```diff
@@ -196,10 +196,10 @@
           "mdn_url": "https://developer.mozilla.org/docs/Web/API/RTCTrackEvent/transceiver",
           "support": {
             "chrome": {
-              "version_added": "56"
+              "version_added": "72"
```
Comment (jpmedley) [inline api/RTCTrackEvent.json]: 69
https://storage.googleapis.com/chromium-find-releases-static/842.html#842893e6fdc0d81da782035ae314433d5c91df20

File: api/RTCTrackEventInit.json [hunk 6]
```diff
@@ -196,10 +196,10 @@
           "mdn_url": "https://developer.mozilla.org/docs/Web/API/RTCTrackEventInit/transceiver",
           "support": {
             "chrome": {
-              "version_added": "56"
+              "version_added": "72"
```
Comment (jpmedley) [inline api/RTCTrackEventInit.json]: 69
https://storage.googleapis.com/chromium-find-releases-static/842.html#842893e6fdc0d81da782035ae314433d5c91df20

**Comment by ddbeck** [comment]
Hi @a2sheppy, do you plan to revisit this soon? #5824 overlaps with this, so I didn't want to move ahead with that, if you were planning to follow up on this soon. Thanks!

File: api/RTCPeerConnection.json (hunk 1 above)
Comment (a2sheppy) [reply api/RTCPeerConnection.json]: Hm, I could have sworn it was disabled or something. OK, will fix.

**Pull request merged**